### PR TITLE
Contact sav

### PR DESCRIPTION
**Pull request by laurentmahieu**

### CODE REVIEW

**Review by BlandineM — APPROVED** [review]

File: frontend/src/components/HomePage.jsx [hunk 1]
```diff
@@ -6,15 +6,19 @@ import NavBar from "./NavBar";
...
   return (
     <div>
-      <CarCard />
       <Switch>
-        <Route path="/historic" component={Historic}></Route>
-        <Route exact path="/" component={ListCar}></Route>
-        <Route path="/intervention/:family" component={Intervention}></Route>
+        <Route path="/contact" component={ContactUs}></Route>
+        <CarCard />
+        <Switch>
```
Comment (LoicBrassart) [inline frontend/src/components/HomePage.jsx:17]: I might be wrong, but I don't think this is correct
We already discussed the idea to have several `<Switch>` in your app, and I can see the benefit (pages without header/footer, was it ?), but I really don't see what you're trying to accomplish here ?

**Review by LoicBrassart — COMMENTED** [review]
Questions to answer

File: frontend/src/components/HomePage.jsx (hunk 1 above)
Comment (laurentmahieu) [reply frontend/src/components/HomePage.jsx:17]: I don't know if that's a bad or a good practice..
but it's working like I want, maybe it should not ? 
what I want : 
the header is a car card with vehicle informations. useless for my page.
my page is a form to contact admin. 
and I want to keep the navbar, if I want to go back or anywhere I want to go without send message.

Comment (laurentmahieu) [reply frontend/src/components/HomePage.jsx:17]: maybe it's better to create my form the page login/signup withtout this switch.
and call under my form the component navbar?

**Review by LoicBrassart — APPROVED** [review]
Not so sure I saw no Evil. I'll let you through with the benefit of the doubt, though !

**Pull request merged**